test/install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if ! ps -ef | grep qemu-system-x86_64 | grep -v grep
+then
+  echo "VMs are not running, please run spin-up-vms.sh first."
+  exit 1
+fi
+
+echo "VMs are running."
+
+echo "Running Terraform to generate inventory and configuration"
+terraform init
+terraform apply -auto-approve
+if [ $? -ne 0 ]; then
```

Comment:
Put "; then" on a seperate line.
```suggestion
if [ $? -ne 0 ]
then
```